### ISSUE TITLE
Errors when attempting to save rules?

### ISSUE THREAD

**Issue by bryangingechen** [sorz/accept-language-per-site#10]
Hi, I just installed the extension in Firefox 121.0 (on macOS 14.2.1) and it didn't seem to be working, so I tried to investigate by "inspecting" the add-on in Firefox to open the dev tools.

With the example rule given in the README (`host: "*.example.com"`, `language: "en"`), I clicked Save in the preferences and got the following output in the "Accept-Language per site" console:

<img width="1368" alt="Screenshot 2024-01-06 at 4 57 22 PM" src="https://github.com/sorz/accept-language-per-site/assets/5209952/c126bd2a-5225-4d66-bfc6-a11be925d926">

Basically, "Error: urlFilter should not start with '||*'" is thrown when the rules are assigned.

I tried changing the `host` to just `example.com` and then tried saving, but then I got "Error: An unexpected error occurred" instead at the same point.

I even tried disabling all my other extensions, but I still got the same errors.

Let me know if there's anything else I can do to help.

**edit**: I just tested this on my Windows 10 machine (with Firefox 121.0) and got the same errors.

**Comment by sorz** [comment]
Thanks! Information you provide are very helpful!

I will release a new version soon.
